pkg/provider/aws/resources/rds.go

```diff
@@ -107,6 +107,85 @@ type (
 	}
 )
 
+type RdsInstanceCreateParams struct {
+	AppName string
+	Refs    []core.AnnotationKey
+	Name    string
+}
+
+// Create takes in an all necessary parameters to generate the RdsInstance name and ensure that the RdsInstance is correlated to the constructs which required its creation.
+//
+// This method will also create dependent resources which are necessary for functionality. Those resources are:
```

Comment:
I'm also not sure the maintenance overhead of listing these resources is worth it. It's relatively trivial to look through the function to see what resources it creates. Especially when this is listing transitive dependencies which may change underneath it.

Might be valuable for complex "<this> set is created based on <x> or <that> set is created based on <y>" but IMO not in these cases.

Reply:
fine by me